doc/embedded/supported_boards.md

```diff
@@ -132,8 +132,8 @@ the new architecture added to this table.
 ### CORTEXA76
 
 ```
--DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/bin/aarch64-buildroot-linux-gnueabihf-
--DCMAKE_SYSROOT=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/aarch64-buildroot-linux-gnueabihf/sysroot
+-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/bin/aarch64-buildroot-linux-gnu-
+-DCMAKE_SYSROOT=/path/to/bootlin/toolchain/aarch64--glibc--stable-2024.02-1/aarch64-buildroot-linux-gnu/sysroot
```

Comment:
Any reason for this change ?

Reply:
It seems like it was a typo in the original; the `gnueabihf` prefix doesn't exist:

```
$ ls /opt/bootlin/aarch64--glibc--stable-2024.02-1/bin/aarch64-buildroot-linux-*-g++
/opt/bootlin/aarch64--glibc--stable-2024.02-1/bin/aarch64-buildroot-linux-gnu-g++
```

This may be an issue with some of the other boards too; I didn't check fully.